index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+
+    <script src="https://kit.fontawesome.com/ccd237d1ae.js" crossorigin="anonymous"></script>
+
+        <title>Sara Mohamoud</title>
+
+    </head>
+
+    <body>
+        <header>
+
+            <div id="logo">
+
+                <span style=color:#DB939A; class="letter"> S </span>
```

Comment:
Separation of concerns. When we separate our HTML and CSS into separate files, it's usually better to stick to that separation throughout the code, rather than having some inline CSS and some imported CSS.